tools/pypsa/features.yaml

```diff
@@ -343,35 +439,41 @@ features:
     resampling:
       value: y
       source:
+        - https://docs.pypsa.org/latest/examples/time-series-aggregation/
         - https://github.com/PyPSA/PyPSA/blob/c5346226ad01de2a9157f8aa198cd14e9fbcfa92/pypsa/common.py#L771
     typical_periods:
-      source: []
-      value: n
+      source:
+        - https://docs.pypsa.org/latest/examples/time-series-aggregation/
+      value: y
   transmission__limits:
     gtc:
-      source: []
-      value: n
+      source:
+        - https://docs.pypsa.org/latest/user-guide/components/lines/
+      value: y
     phase_angle:
-      source: []
-      value: n
+      source:
+        - https://docs.pypsa.org/latest/user-guide/components/lines/
+      value: dev
     ptdf:
       value: y
       source:
-        - https://pypsa.readthedocs.io/en/latest/api/_source/pypsa.SubNetwork.calculate_PTDF.html
+        - https://docs.pypsa.org/latest/api/networks/subnetwork/#pypsa.SubNetwork.calculate_PTDF
     thermal:
       source: []
       value: n
   transmission__opf:
     ac_opf:
-      source: []
-      value: n
+      source:
+        - https://docs.pypsa.org/latest/user-guide/linear-power-flow/
+      value: y
     dc_opf:
-      source: []
-      value: n
+      source:
+        - https://docs.pypsa.org/latest/user-guide/linear-power-flow/
+      value: y
     sec_opf:
       value: y
       source:
-       - https://pypsa.readthedocs.io/en/latest/examples/scigrid-sclopf.html
+        - https://docs.pypsa.org/latest/user-guide/optimization/contingencies/#n-1-security-constraints-for-sclopf
     transport:
       source: []
       value: n
```

Comment:
This is the typical "power flow" for `links`, I believe. I might, as with `coupled` and `ed` above, delete this as it is a baseline for any energy system modelling tool that is spatially disaggregated. For now, set it to `y`